source/includes/hourly-ride-api.md

```diff
@@ -0,0 +1,888 @@
+| originLatitude            | number  | Pickup location latitude in degrees. Required if `originType` is set to `coordinates` or omitted.                                                                                                                                                                                                                     |
+| originLongitude           | number  | Pickup location longitude in degrees. Required if `originType` is set to `coordinates` or omitted.                                                                                                                                                                                                                    |
+| origin                    | string  | IATA airport code of the pickup airport. Required if `originType` is set to `iata`.                                                                                                                                                                                                                                   |
+| departureTime             | integer | Departure time as a UNIX epoch timestamp in seconds. This timestamp should be calculated from the local departure time and converted to UTC.
+| hoursRented               | integer | Duration in hours (1-24). Required. The service includes 20 km of travel per hour.                                                                                                                                                                                                                                    |
+| passengersCount           | integer | Total number of passengers to transport (adults and children). Must be between 1 and 7.                                                                                                                                                                                                                               |
+| childrenCount             | integer | Optional. Specifies the number of children in the group.                                                                                                                                                                                                                                                              |
+| includeNonEnglishSpeaking | boolean | Optional. Defaults to `true`. When set to `false`, no hourly ride options with non-English-speaking drivers will be included.                                                                                                                                                                                         |
```

Comment:
As we are not going to include the Lite service, the `includeNonEnglishSpeaking` parameter doesn't make sense and should be deleted.